odd-platform-api/build.gradle

```diff
@@ -95,6 +96,29 @@ jib {
     allowInsecureRegistries = false
 }
 
+checkstyle {
+    configFile = project(':').file('config/checkstyle/checkstyle.xml')
+    configProperties = [ "suppressionFile" : project(':').file('config/checkstyle/suppressions.xml')]
+    ignoreFailures = false
+    maxWarnings = 0
+}
+
+task buildlessCodestyleCheck(type: Checkstyle) {
+    configFile = project(':').file('config/checkstyle/checkstyle.xml')
+    source 'src/main/java'
```

Comment:
This won't run checks for the test classes.

Reply:
Fixed